shell/shared/grain.js

```diff
@@ -1314,8 +1314,12 @@ if (Meteor.isClient) {
             // Store apitoken id1 and template in session storage in the offer
             // template namespace under key id2.
             var key = "offerTemplate" + id2;
+            var grain = getActiveGrain(globalGrains.get());
+            var grainTitle = grain.title();
+            var slug = grainTitle.toLowerCase().trim().replace(/[^\w ]+/g,'').replace(/ +/g,'-');
             var renderedTemplate = template.replace(/\$API_TOKEN/g, tokenId)
-                                           .replace(/\$API_HOST/g, makeWildcardHost("api"));
+                                           .replace(/\$API_HOST/g, makeWildcardHost("api"))
+                                           .replace(/\$GRAIN_SLUG/g, slug);
```

Comment:
I'd prefer this be called `GRAIN_TITLE_SLUG` rather than `GRAIN_SLUG` to make it slightly more precise.